tests/x11/nis_client.pm

```diff
@@ -98,7 +98,10 @@ sub nfs_shares_tab {
     send_key 'alt-o';          # OK
     assert_screen 'nis-client-fw-opened';
     send_key 'alt-f';          # finish
-    if (check_screen 'disable_auto_login_popup') { send_key 'ret' }
+    if (is_opensuse) {
+        assert_screen 'disable_auto_login_popup';
+        send_key 'ret';
```

Comment:
Is pressing enter good enough? Or it would be better to be more explicit about pressing alt-n/y?
LGTM other that that.

Reply:
I guess just increasing timeout for check_screen might help, as it seems like it takes some time for the popup to be appeared: https://openqa.opensuse.org/tests/1225940#step/nis_client/61

Reply:
Pressing enter works. Anyway we are not asserting an ything afterwards :D (see https://progress.opensuse.org/issues/65349). I thought of the timeout, but if it still fails it's not obvious what went wrong. And it will fail as soon as there is a UI change. I did not chose it first because that popup will come to SLE son or late, but well, then it will be very easy to remove that check.

Reply:
fixed expectations in general are fine, but in this case I'd go with increasing timeout, because we will need to remove `is_opensuse` afterwards, if it is checking for enough time, we should see screenshots helping to understand the issue. When using `check_screen` it is more needed (than when using and assertion as in your current code) to not use 'ret' as we could hit any other button afterwards in case the check would not match anything.